### DISCUSSION TITLE
Playbook config and working support

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#7779] [Unsupported Versions] [answered]
Hi friends,
Anyone help me in how to use playbooks and how can it generate alert any demo or steps or snaps shot that can help me for POC..
I have make all available playbooks active but didn't get any kind of alert yet ..how can i generate the dummy alert or something like that.
Any steps and configuration that can help me ..ASAP

Thanks

**Comment by defensivedepth** [comment, marked as answer]
@usmanaa10 I see that you already asked this question less than 24 hours ago - https://github.com/Security-Onion-Solutions/securityonion/discussions/7767

Please do not create multiple Discussions for the same issue.
